modules/nested-elements/assets/js/editor/nested-repeater/component.js

```diff
@@ -15,6 +15,25 @@ export default class Component extends $e.modules.ComponentBase {
 		super.registerAPI();
 
 		elementor.addControlView( 'nested-elements-repeater', RepeaterControl );
+
+		// Disabling 'delete' from context menu of building blocks containers.
+		const callback = ( groups, view ) => {
```

Comment:
Extract to a method like `disableContextMenuDelete`, also, check if it's the first nested child